Documentation/ApiOverview/Backend/PageTree.rst

```diff
@@ -0,0 +1,42 @@
+:navigation-title: Page tree
+..  include:: /Includes.rst.txt
+..  _page-tree:
+
+==================================
+The page tree in the TYPO3 backend
+==================================
+
+The **page tree** is a hierarchical structure that represents pages and their
+subpages on a website, allowing editors and integrators to easily organize and
+manage content and navigation.
+
+It is displayed on the left of backend modules with
+:ref:`navigationComponent <t3coreapi:confval-backend-module-navigationcomponent>`
+set to `'@typo3/backend/tree/page-tree-element'`.
+
+Usage of the page tree is described in
+`Getting Started Tutorial, Page tree <https://docs.typo3.org/permalink/t3start:page-tree>`_.
+
+The page tree can also be navigated via `Keyboard commands (Tutorial for
+Editors) <https://docs.typo3.org/permalink/t3editors:keyboard-commands>`_.
+
+..  contents::
```

Comment:
I miss a separator here. Last sentence is about navigation followed by three links, but the last two links open section about event listeners and user TSconfig

```suggestion
..  contents:: Table of contents
```